### DISCUSSION TITLE
Bug: Side tab don't show the sticky tabs and + sign when you have many tabs open

### DISCUSSION THREAD

**Discussion by mr-cheffy** [zen-browser/desktop#776] [Ideas]
### What happened?

If you have many tabs opened beyond the size of the side panel, you can't access the plus sign to open a new tab nor the sticky one, unlike the normal behavior in firefox. Right now, if I need to access any of these, I have to scroll through my hundreds of tabs to access them, defeating the purpose to have them.
![Screenshot_20240825_030418](https://github.com/user-attachments/assets/3839f678-bc3d-4e72-a21a-8e0eb69f2c52)

### Reproducible?

- [ ] I have checked that this issue cannot be reproduced on Mozilla Firefox.

### Version

1.0.0.a28

### Severity impact

Enhancement

### What platform are you seeing the problem on?

Linux, Windows

 
 ~ _Created automatically from issue: https://github.com/zen-browser/desktop/issues/646_

**Comment by ricardofrancoli** [comment]
I believe a good solution would be moving it out of the tabs container and leave it fixed either at the top, just below the workspace button, or at the bottom, just above the "List all tabs" button or the subsection with the profile, settings, etc.

I'm happy to work on this if this is something we decide to do. Also open to hear suggestions or alternative implementation ideas!

## Fixed at top option:
![image](https://github.com/user-attachments/assets/8262412b-f990-43d9-bc57-770fb5ec3e73)
![image](https://github.com/user-attachments/assets/94f73b0c-0c25-46f2-9b7c-58ab8d24d2f0)

## Fixed at bottom option:
![image](https://github.com/user-attachments/assets/be31d09f-715d-48ae-8250-581a25b09962)
![image](https://github.com/user-attachments/assets/ce1bb7ff-f8a8-4665-ab83-f8b5fa2f89fe)
